lib/Service/AssistantService.php

```diff
@@ -86,6 +88,36 @@ public function __construct(
 		private IConfig $config,
 		private IShareManager $shareManager,
 	) {
+		$this->informationSources = [
+			'ask_context_chat' => $this->l10n->t('Context Chat'),
+			'transcribe_file' => $this->l10n->t('Assistant File Transcription'),
+			'generate_document' => $this->l10n->t('Assistant Document Generation'),
+			'list_calendars' => $this->l10n->t('Nextcloud Calendar'),
+			'schedule_event' => $this->l10n->t('Nextcloud Calendar'),
+			'find_free_time_slot_in_calendar' => $this->l10n->t('Nextcloud Calendar'),
+			'add_task' => $this->l10n->t('Nextcloud Tasks'),
+			'find_details_of_current_user' => $this->l10n->t('Nextcloud User Profile'),
+			'list_decks' => $this->l10n->t('Nextcloud Deck'),
+			'add_card' => $this->l10n->t('Nextcloud Deck'),
+			'get_coordinates_for_address' => $this->l10n->t('Openstreetmap'),
+			'get_current_weather_for_coordinates' => $this->l10n->t('Norwegian Meteorological Institute Weather Forecast'),
+			'get_public_transport_route_for_coordinates,' => $this->l10n->t('HERE Public Transport API'),
+			'get_osm_route,' => $this->l10n->t('Openstreetmap'),
+			'get_osm_link,' => $this->l10n->t('Openstreetmap'),
```

Comment:
Nitpick: It's spelled [OpenStreetMap](https://en.wikipedia.org/wiki/OpenStreetMap) :)